home/.vimrc

```diff
@@ -97,7 +97,7 @@ map \ :NERDTreeToggle<CR>
 
 map <Leader>p gg:s/\s*$<CR>/\d\+-<CR>ywggA [#<esc>pa]<esc>
 nmap <Leader>l iLorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.<esc>
-
+nmap <Leader>skip gg2o<esc>i[skip ci]<esc>
```

Comment:
Why'd this change from `<Leader>c`?  This is long enough I'd forget and just `o<cr>[skip ci]` by hand...